pages/configuring_ide_netbeans.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Configuring your IDE
+permalink: /configuring-ide-netbeans/
+redirect_from:
+  - /configuring_ide.html
+sitemap:
+    priority: 0.7
+    lastmod: 2015-05-22T18:40:00-00:00
+---
+
+# <i class="fa fa-keyboard-o"></i> Configuring Netbeans
+
+Importing your JHipster application in Netbeans is straightforward.
+
+## 1. Open your project
+
+- Select File -> Open Project
+- Browse and select your project's folder
+
+![Open]({{ site.url }}/images/configuring_ide_netbeans_1.png)
+
+**That's All** : To run your project, just do a right-click on the Application class and select "Run File".
```

Comment:
Maybe you could add that the "Application" class is at the root of the package hierarchy